data-prepper-core/src/main/java/com/amazon/dataprepper/parser/PipelineConfigurationValidator.java

```diff
@@ -123,6 +123,7 @@ private static void validatePipelineAttributeName(final String pipelineAttribute
 
     /**
      * Validates for orphan pipeline configurations causing ambiguous execution model.
+     * TODO: Should this be removed? (unused code)
```

Comment:
We intentionally left this as there was an unfinished discussion about allowing orphan pipelines, think we can leave this for now or finish the discussion if we allow or do not allow orphan pipelines and decide based on that

Reply:
Thanks for the info. Will leave it for now to unblock merging - we can come back when cleaning up other TODOs.